examples/common.cpp

```diff
@@ -1,15 +1,22 @@
 #include "common.h"
 
 #include <cassert>
+#include <iostream>
```

Comment:
For cleaner interoperability between C and C++, this repo doesn't use C++'s iostreams. You'll find the repo uses `fprintf(stderr` as opposed to `std::cerr`.
```suggestion
```